src/components/Header/MenuAccountDropdown/LogOut.js

```diff
@@ -9,8 +9,8 @@ import style from './dropdown.scss'
 const cx = cn.bind(style)
 
 const LogOut = ({ logout }) => (
-  <li key="action-logout" className={cx('action')}>
-    <button type="button" onClick={() => logout()}>
+  <li key="action-logout" className={cx('action')} onClick={logout}>
```

Comment:
Why did you move onClick to `<li>`?

Reply:
@andre-meyer because <button /> is too small :(